packages/polkadot/package.json

```
@@ -0,0 +1,57 @@
+{
+    "name": "@openguild-labs/agent-kit-polkadot",
```

```suggestion
    "name": "@dot-agent-kit/core",
```

Recommend renaming the package to `core` instead of `polkadot`. Because the library is limited to only Polkadot